docs/docsite/rst/intro_inventory.rst

```diff
@@ -118,6 +118,8 @@ Variables can also be applied to an entire group at once::
    ntp_server=ntp.atlanta.example.com
    proxy=proxy.atlanta.example.com
 
+Be aware that this is only a convenient way to apply variables to multiple hosts at once, even though you can target hosts by group, variables are always flattened to the host level before a play is executed.
```

Comment:
Please change the comma after the word "once" to a semicolon or split this into two sentences.